### ISSUE TITLE
Network crashed after Failed sending broadcast message:Mailbox has closed

### ISSUE THREAD

**Issue by ailisp** [nearprotocol/nearcore#1938]
Happens in 100 node test. Almost at same time 05:17:14Z all nodes crash with the following output: there was no memory and cpu issue at time of exit.

```
[2020-01-10T05:17:14Z INFO  near_client::info] #    6760 269PCKpU5VKoZT1nXfGwpSyAnaeE8nzQzCYh7NLdNbDp -/1 40/19/40 peer ⬇ 59.6kiB/s ⬆ 35.6kiB/s 1.00 bps 0 gas/s CPU: 1%, Mem: 141.7 MiB
[2020-01-10T05:17:24Z INFO  near_client::info] #    6769 6Bp6HzQow94KZopA4fMRPe9cHAjiXC5qNVXK4QSBAGzt -/1 40/19/40 peer ⬇ 59.6kiB/s ⬆ 35.6kiB/s 0.90 bps 0 gas/s CPU: 1%, Mem: 141.7 MiB
[2020-01-10T05:17:30Z ERROR near_network::peer_manager] Failed sending broadcast message: Mailbox has closed
[2020-01-10T05:17:30Z ERROR near_network::peer_manager] Failed sending broadcast message: Mailbox has closed
thread 'main' panicked at 'Decoder error: Os { code: 104, kind: ConnectionReset, message: "Connection reset by peer" }', src/libcore/result.rs:1165:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
thread 'main' panicked at 'Decoder error: Os { code: 104, kind: ConnectionReset, message: "Connection reset by peer" }', src/libcore/result.rs:1165:5
[2020-01-10T05:17:30Z ERROR near_network::peer_manager] Failed sending broadcast message: Mailbox has closed
[2020-01-10T05:17:30Z ERROR near_network::peer_manager] Failed sending broadcast message: Mailbox has closed
[2020-01-10T05:17:30Z ERROR near_network::peer_manager] Failed sending broadcast message: Mailbox has closed
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "Non-zero exit code: 1" }', src/libcore/result.rs:1165:5
```

node logs: https://transfer.sh/SNod0/log.tar.gz

**Comment by bowenwang1996** [comment]
Yeah there is some actix issue in the network code

**Comment by ailisp** [comment]
Rename it to highlight the issue, it's all nodes in network crashed rather than some of nodes.

**Comment by mfornet** [comment]
I've seen this recently and I think it is related with the update of future. I'll investigate further.

**Comment by mikhailOK** [comment]
#1954

**Comment by mfornet** [comment]
Solved in #1954 (See follow up in #1988).
